wolfssl/wolfcrypt/settings.h

```diff
@@ -169,6 +169,19 @@
 /* Uncomment next line if building for using XILINX */
 /* #define WOLFSSL_XILINX */
 
+/* Uncomment next four lines for using WICED Studio. */
+/* #if !defined(WOLFSSL_WICED) */
+/*    #define WOLFSSL_WICED    */
+/* #endif  */
+
+#if defined(WOLFSSL_WICED)
```

Comment:
This section should be down below. Your WICED project must define the pre-processor macro `WOLFSSL_USER_SETTINGS`.